src/lang/en-US/Workflows.ts

```diff
@@ -63,7 +63,7 @@ export const Workflows: Record<string, string> = {
 
     'workflows.collectionSelector.footer.backfilled': `backfilled: {calculatedCount}`,
     'workflows.collectionSelector.footer.backfilled.all': `all backfilled`,
-    'workflows.collectionSelector.footer.backfilled.empty': ` `,
+    'workflows.collectionSelector.footer.backfilled.empty': `-`,
```

Comment:
Watching folks use stuff they seem to stumble over the small blank space at the bottom of bindings. So just adding a small visual marker to potentially make it more clear this is a "intentionally left blank" kind of thing.

Reply:
Due to feedback.